x/group/client/cli/util.go

```diff
@@ -66,6 +66,8 @@ type Proposal struct {
 	Messages  []json.RawMessage `json:"messages,omitempty"`
 	Metadata  string            `json:"metadata"`
 	Proposers []string          `json:"proposers,omitempty"`
+	Title     string            `json:"title"`
```

Comment:
In `cli/util_test.go`, could you add the two fields and test it is correctly parsed?